charts/elasticsearch-curator/values.yaml

```diff
@@ -0,0 +1,10 @@
+image:
+  repository: bobrik/curator
+  tag: 3.5.1
+  pullPolicy: IfNotPresent
+schedule: "0 1 * * *"
```

Comment:
Even if I roughly know how the cron fields work they always puzzle me. I wonder if a comment would help, e.g. "Run every day at 1:00am". Or just ignore me.